beta/docker/README.md

```diff
@@ -0,0 +1,231 @@
+- [`scim.env`](./scim.env): an environment file used to configure SCIM bridge
+
+## Overview
+
+The open source Docker Engine tooling can be used to deploy 1Password SCIM Bridge on any supported Linux distribution. A single-node swarm is sufficient. This example assumes that the Linux server is exposed directly to the public internet with Docker acting as a reverse proxy to the SCIM bridge container. Other configurations are possible, but not documented in this deployment example.
+
+## Prerequisites
+
+- Linux VM or bare metal server supporting Docker
+- Docker Engine (v20.10.3+) (see [Docker Engine installation overview](https://docs.docker.com/engine/install/#server))
+- public DNS record pointing to the Linux server
+
+## Get started
+
+1. Create a public DNS record that points to the public IP address of the Linux server for your SCIM bridge. For example, `scim.example.com`.
+
+2. [Install Docker Engine](https://docs.docker.com/engine/install/#server) on the server (follow the Server instructions; Docker Desktop is not needed). We also recommend following the [post-install steps](https://docs.docker.com/engine/install/linux-postinstall/) as noted in the documentation to enable running Docker as a non-root user and ensure that Docker Engine starts when the Linux system boots.
+
+3. All following steps can be run directly on the Linux server, or by connecting remotely using the `--host` parameter or `docker context`. If you are already using SSH to connect to the Linux server, you can create and use a Docker context on your local computer that is connected to your 1Password account (these commands require Docker Engine or Docker Desktop to be installed on your computer).
+
+   *Example commands:*
+
+   ```sh
+   docker context create op-scim-bridge \
+       --description "1Password SCIM Bridge Docker host" \
+       --docker host=ssh://user@scim.example.com
+   docker context use op-scim-bridge
+   ```
+
+   Replace `user` with the appropriate username for your Linux server and `scim.example.com` with its host name before running the above commands.
+
+4. Initialize a swarm. Run this command:
+
+   ```sh
+   docker swarm init # --advertise-addr 192.0.2.1
+   ```
+
+   > **Note**
+   >
+   > If Docker returns an error about multiple network interfaces, uncommment the `--advertise-addr` parameter and replace the example IP (`192.0.2.1`) with the public IP address of the Linux server.
+
+5. Clone this repository and switch to this directory:
+
+    ```sh
+    git clone https://github.com/1Password/scim-examples.git
+    cd ./scim-examples/beta/docker
+    ```
+
+6. Save your `scimsession` file to the working directory.
+
+   > **Note**
+   >
+   > If you are deploying SCIM bridge directly on the Linux server, you will need to transfer the `scimsession` file
+   > from your local machine that is signed in to your 1Password account, for example:
```

Comment:
> from your local machine that is signed in to your 1Password account, for example:
Another reference to "signed in to your 1Password account" that I'm not sure I understand the significance of. 

Maybe we need to clarify this at the beginning of the docs so that what this means, and it's significance, can be better understood throughout? 

In theory this _could_ be accomplished by someone who has the scimsession file and isn't yet a member of the account the SCIM bridge is associated with, so theoretically as long as you have the credentials you don't even need to be signed in nor have an account.